google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/StorageImpl.java

```diff
@@ -63,6 +63,7 @@
 import com.google.common.io.BaseEncoding;
 import com.google.common.net.UrlEscapers;
 import com.google.common.primitives.Ints;
+import com.google.cloud.storage.HmacKey.HmacKeyMetadata;
```

Comment:
I don't think this is necessary.

Reply:
It is, would need "HmacKey.HmacKeyMetadata" without it

Reply:
ah, gotcha. Thanks for clarifying!